packages/react-native-sdk/src/components/Call/CallContent/CallContent.tsx

```diff
@@ -201,67 +197,82 @@ export const CallContent = ({
     supportedReactions,
   };
 
-  const landscapeStyles: ViewStyle = {
-    flexDirection: landscape ? 'row' : 'column',
-  };
-
   return (
     <>
       {!disablePictureInPicture && (
         <RTCViewPipIOS
           includeLocalParticipantVideo={iOSPiPIncludeLocalParticipantVideo}
         />
       )}
-      <View style={[styles.container, landscapeStyles, callContent.container]}>
-        <View style={[styles.container, callContent.callParticipantsContainer]}>
+      <View style={styles.mainContainer}>
+        <View style={styles.unsafeArea} />
+        <SafeAreaView style={[styles.safeArea, callContent.container]}>
```

Comment:
```suggestion
        <SafeAreaView style={[styles.safeArea, callContent.container]}>
```
Please do not use SafeAreaView inside the SDK

Reply:
also check and remove anywhere else where you use it, insets from theme should be applied as padding